CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contribution Guide for PolyMath
+
+This file is currently not complete but will be improve step by step.
+
+# Release management
+
+This project use semantic versionning to define the releases. This mean that each stable release of the project will get associate a version number of the form `vX.Y.Z`. 
```

Comment:
May I suggest:

This project uses semantic versioning to define the releases, meaning that each stable release of the project will be assigned a version number of the form `vX.Y.Z`.